code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -233,10 +237,15 @@
 
 /mob/living/simple_animal/hostile/carp/advanced
 	name = "advanced space carp"
-	desc = "A ferocious, fang-bearing creature that resembles a fish. This one seems to be self-aware."
+	desc = "A ferocious, fang-bearing creature that resembles a fish."
 	maxHealth = 40
 	health = 40
 	gold_core_spawnable = NO_SPAWN
 	obj_damage = 15
 
+/mob/living/simple_animal/hostile/carp/advanced/examine(mob/user)
+	. = ..()
+	if(!isnull(client))
+		. += "<span class='notice'>This one seems to be self-aware.</span>"
```

Comment:
I think this should be at base carp.

Reply:
This can be applied to the base carp yes, but then we can turn around and say "I think this should be at the base simple_animal". My thought behind it is this examine is clearly wrong at times, might as well make it accurate. Maybe we don't want all carps or simple_animals to "tattle" if there is a player controlling them?